fnet/network.go

```diff
@@ -26,3 +27,19 @@ func NormalizePort(port string) string {
 	}
 	return ":" + port
 }
+
+// GRPCDestination parses dest and returns dest:port based on dest type
+// being a hostname, IPv4 or IPv6 address.
+func GRPCDestination(dest, port string) string {
```

Comment:
Need to either move this in fgrpc/ or rename it to something more generic
`NormalizeHostPort()` maybe ?

also rename "port" param to "defaultPort" for clarity that it won't always be used